style.css

```diff
@@ -0,0 +1,578 @@
+    color: #ec5242;
+    font-size: 14px;
+    text-align: center;
+    text-shadow: #272a31 2px 2px 3px;
+  }
+
+  .main {
+    background-color: rgb(39, 42, 50);
+    background-image: url(./images/pattern_bg.png);
+    background-repeat: repeat;
+    color: #d3d3d3;
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .main h3 {
+    margin-top: 30px;
+  }
+
+  .main hr {
+    margin: 15px 0;
+    border: #ec5242 1px solid;
+    width: 10%;
+  }
+
+  .container-main {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .list-item {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    background-color: rgb(211, 211, 211, 0.15);
+    margin: 10px;
+    padding: 10px;
+    width: 90%;
+    align-items: center;
+  }
+
+  .list-item > li {
+    margin: 2%;
+  }
+
+  .text1 {
+    font-size: 12px;
+    color: #ec5242;
+  }
+
+  .text2 {
+    font-size: 12px;
+    color: #d3d3d3;
+    font-weight: 200;
+  }
+
+  .text3 {
+    font-size: 12px;
+    color: rgb(39, 42, 49, 65);
+    font-weight: 200;
+  }
+
+  #joinBtn {
+    background-color: #ec5242;
+    font-size: 16px;
+    color: #d3d3d3;
+    border: none;
+    padding: 20px;
+    margin: 30px;
+  }
+
+  #program-link {
+    display: none;
+  }
+
+  .article {
+    color: #272a31;
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .article h3 {
+    margin-top: 30px;
+  }
+
+  .article hr {
+    margin: 15px 0;
+    border: #ec5242 1px solid;
+    width: 10%;
+  }
+
+  .list-item-speaker {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    margin: 10px;
+    padding: 10px;
+    align-items: center;
+  }
+
+  .list-item-speaker ul {
+    margin: 2%;
+    list-style: none;
+  }
+
+  .list-item-speaker li {
+    margin: 3%;
+  }
+
+  .artPartner {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color: #f4f3f3;
+    background-color: rgb(39, 42, 49, 0.9);
+  }
+
+  .artPartner h3 {
+    margin-top: 30px;
+    font-size: 18px;
+  }
+
+  .artPartner hr {
+    margin: 15px 0;
+    border: #ec5242 1px solid;
+    width: 10%;
+  }
+
+  .sponsors {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding: 0 20px;
+  }
+
+  footer img {
+    filter: invert();
+    margin: 30px;
+  }
+
+  .sponsors img {
+    width: 30%;
+    height: auto;
+    margin: 10px;
+  }
+
+  footer {
+    display: flex;
+    justify-content: center;
+    font-size: 12px;
+    color: #272a31;
+    padding: 30px;
+    text-align: center;
+    align-items: center;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .cocogoose {
+    font-family: 'COCOGOOSE', serif;
+    font-style: normal;
+    font-weight: 400;
+  }
+
+  .lato {
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+  }
```

Comment:
[OPTIONAL] I would recommend reducing the weight and the size of the fonts in the desktop version to improve your UI  design:
- Your design:
<img width="959" alt="image" src="https://user-images.githubusercontent.com/77190203/163663251-cfba05bd-f0e7-486b-80d9-aca5f3d92aed.png">

- Original design:
<img width="449" alt="image" src="https://user-images.githubusercontent.com/77190203/163663276-830a38a9-f74d-4837-94f0-c55c2dbd5e0c.png">